applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -7,14 +7,37 @@ toc::[]
 
 The *Developer* perspective in the web console provides you the following options from the *+Add* view to create applications and associated services and deploy them on {product-title}:
 
-* *From Git*: Use this option to import an existing codebase in a Git repository to create, build, and deploy an application on {product-title}.
-* *Container Image*: Use existing images from an image stream or registry to deploy it on to {product-title}.
-* *From Dockerfile*: Import a dockerfile from your Git repository to build and deploy an application.
-* *YAML*: Use the editor to add YAML or JSON definitions to create and modify resources.
-* *From Catalog*: Explore the *Developer Catalog* to select the required applications, services, or source to image builders and add it to your project.
-* *Database*: See the *Developer Catalog* to select the required database service and add it to your application.
-* *Operator Backed*: Explore the *Developer Catalog* to select and deploy the required Operator-managed service.
-* *Helm Chart*: Explore the *Developer Catalog* to select the required Helm chart to simplify deployment of applications and services.
+* *Getting Started resources*: Use these resources to help you get started with the Console. You can choose to hide the header using the *Options* menu {kebab}.
```

Comment:
On the page it is called "Getting started resources" with lower s at started. Should we call it "Developer Console" at the end? Up to you

```suggestion
* *Getting started resources*: Use these resources to help you get started with Developer Console. You can choose to hide the header using the *Options* menu {kebab}.
```

Reply:
Done!